### DISCUSSION TITLE
Where do comments have their submit button?

### DISCUSSION THREAD

**Discussion by NoobTracker** [ScratchAddons/ScratchAddons#200] [Help]
Now this is weird: Comments don't have a submit button. The post button is just a link to #null. Can anyone tell me how that works? @JeffaloBob I think you already did those weirt out-of-context replies, maybe you can help me.

**Comment by jeffalo** [comment]
first off #200 🎉 but i don't understand what you're saying?

**Comment by retronbv** [comment]
OOH I KNOW!!! in their html xD
but actually no

**Comment by NoobTracker** [comment]
![thatsaproblemiguess](https://user-images.githubusercontent.com/63962365/92262916-1de53d80-eedc-11ea-98f8-d557c3044b0c.PNG)
(for those who don't speak german, Leerraum means empty space)
forms usually have a submit button. With type="submit".

**Comment by jeffalo** [comment]
> OOH I KNOW!!! in their html xD
> but actually no

... what?

**Comment by NoobTracker** [comment]
> 
> 
> OOH I KNOW!!! in their html 

:roll_eyes: I'm glad this smiley exists

**Comment by retronbv** [comment]
>... what?

the submit button is in the html

**Comment by jeffalo** [comment]
> ![thatsaproblemiguess](https://user-images.githubusercontent.com/63962365/92262916-1de53d80-eedc-11ea-98f8-d557c3044b0c.PNG)
> (for those who don't speak german, Leerraum means empty space)
> forms usually have a submit button. With type="submit".

i'm pretty sure scratch's javascript handles comment posting, and the a tag is just there to have some kind of clickable element.

**Comment by NoobTracker** [comment]
But how can they identify that that `a` is clicked?

**Comment by retronbv** [comment]
in the js look for any `a` with the `href` of `#null`

**Comment by jeffalo** [comment]
> But how can they identify that that `a` is clicked?

because javascript?

**Comment by NoobTracker** [comment]
There is no event listed.

**Comment by NoobTracker** [comment]
As you can see at `<textarea>`, Firefox shows events.

**Comment by NoobTracker** [comment]
Can you find out that something was clicked without using an event? @JeffaloBob?

**Comment by GrahamSH-LLK** [comment]
Where is this code?

**Comment by Explosion-Scratch** [comment]
#200 nice!! (I don't speak javascriot much yet so I can't really contribute)

**Comment by NoobTracker** [comment]
> 
> 
> Where is this code?

It's from the comment box on your profile and everywhere else too I guess.

**Comment by GrahamSH-LLK** [comment]
> > Where is this code?
> 
> It's from the comment box on your profile and everywhere else too I guess.

Ah! Profiles are scratchr2. That explains it.

**Comment by NoobTracker** [comment]
Okay, didn't know that there's a difference.

**Comment by GrahamSH-LLK** [comment]
Project pages are Scratch-WWW. Studios and Profiles are scratchr2, the older repo.

**Comment by NoobTracker** [comment]
Okay, I didn't know that the `form`s are different. I know that those pages look different (and I think that r2 looks better).

**Reply by GrahamSH-LLK** [reply]
Um, WHAT???? WWW is so much better! 😜

**Reply by NoobTracker** [reply]
I think that 3.0 looks like a smartphone app and I don't like smartphone apps.

**Reply by Hans5958** [reply]
Agreed. It has too big sizes compared to 2.

**Comment by retronbv** [comment]
on the profile comment button 
![image](https://user-images.githubusercontent.com/49005044/96335282-a01d6180-1045-11eb-9e2a-ad2bc73191d6.png)

**Reply by Hans5958** [reply]
@GrahamSH-LLK, the "Mark as answer" button should be used if NoobTracker thinks that it solves the problem.

**Comment by Explosion-Scratch** [comment]
Mark as answer fight lol
